cgroups/stats.go

```diff
@@ -30,18 +30,21 @@ type CpuStats struct {
 	ThrottlingData ThrottlingData `json:"throttling_data,omitempty"`
 }
 
+type MemoryData struct {
+	Usage    uint64 `json:"usage,omitempty"`
+	MaxUsage uint64 `json:"max_usage,omitempty"`
+	Failcnt  uint64 `json:"failcnt"`
+}
 type MemoryStats struct {
```

Comment:
I agree that the change is preferable, but I don't know what API guarantees Docker makes in the stats API and I wanted to let @crosbymichael know :)